test/postgresql.discover.test.js

```diff
@@ -339,4 +339,14 @@ describe('Discover and build models', function() {
       });
     });
   });
+
+  it('should build a model from discovery with comments', function(done) {
+    db.discoverAndBuildModels('TestGeo', {schema: 'public'}, function(err, schema) {
+      var prop = schema.Testgeo.definition.properties.loc.postgresql;
```

Comment:
Is there any particular reason for scoping the description to `postgresql`? I would personally prefer the description to be LoopBack-wide.

```ts
const prop = schema.Testgeo.definition.properties.loc;
assert.equal(prop.description, 'The location of property of TestGeo');
```

When the description is set directly on the property, then tools like loopback-swagger (powering loopback-component-explorer in LB v3) and `@loopback/repository-json-schema` (in LB v4) will pick it up and use it to describe the property in Swagger/OpenAPI spec.